.github/workflows/ci.yml

```diff
@@ -8,13 +8,34 @@ jobs:
     strategy:
       # The matrix will run all possible combinations in parallel.
       matrix:
-        container: [ ubuntu-latest, macos-latest ]
+        container: [ ubuntu-latest, macos-latest, windows-latest ]
 
     runs-on: ${{ matrix.container }}
 
     steps:
       - uses: actions/checkout@v2
 
+      - name: Set up constants
+        id: constants
+        run: |
+          if [ "$RUNNER_OS" == "Linux" ]; then
+            echo ::set-output name=toit_archive::toit-linux.tar.gz
+            echo '::set-output name=toitvm::${{ github.workspace }}/toit/bin/toit.run'
+            echo '::set-output name=toitpkg::${{ github.workspace }}/build/toitpkg'
+          elif [ "$RUNNER_OS" == "macOS" ]; then
+            echo ::set-output name=toit_archive::toit-macos.tar.gz
+            echo '::set-output name=toitvm::${{ github.workspace }}/toit/bin/toit.run'
+            echo '::set-output name=toitpkg::${{ github.workspace }}/build/toitpkg'
+          elif [ "$RUNNER_OS" == "Windows" ]; then
```

Comment:
again, I would prefer it done in the yaml language instead.

Reply:
As explained in the other PR: we can't do that, since we are referring to the output variables by step-id. Having different steps for the platforms would make that impossible.